src/components/header_menu/HeaderMenuToolbar.js

```diff
@@ -114,7 +114,7 @@ class HeaderMenuToolbar extends React.Component {
                 <MenuItem
                     onClick={this.props.onToggleIsNonPeriodic}>
                     <ListItemIcon><DeviceHubIcon/></ListItemIcon>
-                    Toggle "isNonPeriodic"
+                    Set As "Non-Periodic"
```

Comment:
label of switch changed assuming we want the switch to only work for converting from `periodic -> non-periodic`